src/Tasks/Microsoft.NET.Build.Tasks/ResolveReadyToRunCompilers.cs

```diff
@@ -393,8 +399,9 @@ private bool GetCrossgen2ComponentsPaths(bool version5)
             }
             else
             {
-                // Unknown platform
-                return false;
+                // Generic Unix-like: linux, freebsd, and others. Does not check for arch
```

Comment:
Could you please delete `RuntimeInformation.IsOSPlatform(OSPlatform.Linux)` above?